CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- check-smart.rb: Removed duplicated smartctl runs
+- check-smart.rb: The JSON file is now does not need to exist
+- check-smart.rb: Add support for legacy smartctl output relating to `Available` and `Enabled`
```

Comment:
These two should be under a header of `### Added`

Reply:
I've updated the Changelog with Added, Removed and Changed sub headings